test/unit/math/prim/core/init_threadpool_tbb_late_test.cpp

```diff
@@ -10,12 +10,12 @@
 #include <tbb/task_arena.h>
 
 TEST(intel_tbb_new_late_init, check_status) {
-  const int num_threads = tbb::global_control::default_num_threads();
```

Comment:
`tbb::global_control` class doesn't have `default_num_threads` member. This is fixed here.